### PR TITLE
[PAR-4407] add extendStoreId to payload if it exists

### PR DESCRIPTION
**Pull request by aarondeane-extend**
# Description
- Include the `extendStoreId` from the integration record if it exists (Guidance)

## Ticket
https://helloextend.atlassian.net/browse/PAR-4407

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have deployed and validated my code on a sandbox
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by jomarsantos — DISMISSED** [review]

File: Plugin/Controller/Adminhtml/Integration/SavePlugin.php [hunk 1]
```diff
@@ -192,10 +192,18 @@ private function sendIntegrationToExtend($integrationId, $storeId)
                 'magentoStoreUuid' => $integrationStore->getStoreUuid(),
                 'magentoStoreId' => $storeId,
                 'magentoConsumerKey' => $oauth->getKey(),
-                'storeDomain' => $this->scopeConfig->getValue(
-                    Store::XML_PATH_UNSECURE_BASE_URL,
-                    'store',
-                    $storeId
+                'extendStoreId' => $integrationStore->getExtendStoreUuid(),
+                'storeDomain' => rtrim(
+                    str_replace(
```
Comment (aarondeane-extend) [inline Plugin/Controller/Adminhtml/Integration/SavePlugin.php:197]: this removes the `https://` or `http://` and the `/` from the `storeDomain`

**Review by scott-taylor-extend — DISMISSED** [review]

File: Plugin/Controller/Adminhtml/Integration/SavePlugin.php (hunk 1 above)
Comment (jay-faust-extend) [reply Plugin/Controller/Adminhtml/Integration/SavePlugin.php:197]: What is the metadata builder expecting? I could check but it feels weird doing this with no corresponding metadata builder changes

**Review by milesmcleod — APPROVED** [review]

File: Plugin/Controller/Adminhtml/Integration/SavePlugin.php (hunk 1 above)
Comment (aarondeane-extend) [reply Plugin/Controller/Adminhtml/Integration/SavePlugin.php:197]: yeah not my fave either but the `MetaDataBuilder` is just sending the `storeId` not the `storeDomain`

Comment (jay-faust-extend) [reply Plugin/Controller/Adminhtml/Integration/SavePlugin.php:197]: Then why do we give it the store domain? Or am I not reading it right?

Comment (aarondeane-extend) [reply Plugin/Controller/Adminhtml/Integration/SavePlugin.php:197]: it's in the `data` object here which is just passed along
https://github.com/helloextend/magento-extend-integration/blob/26e43ace7849b5c8f3b2876d1465d95e0b6b652c/Service/Api/MetadataBuilder.php#L41

**Review by jay-faust-extend — APPROVED** [review]

**Review by scott-taylor-extend — APPROVED** [review]

**Pull request merged**